src/utils/hooks/useFetchDepartments.ts

```diff
@@ -7,9 +7,7 @@ import { setNotification } from '../../redux/actions/notificationActions';
 import { NotificationVariant } from '../../types/notification.types';
 import useCancelToken from './useCancelToken';
 
-const useFetchDepartments = (
-  departmentId: string = ''
-): [RecursiveInstitutionUnit[], boolean, (departmentId?: string) => void] => {
+const useFetchDepartments = (departmentId: string = ''): [RecursiveInstitutionUnit[], boolean] => {
```

Comment:
```suggestion
const useFetchDepartments = (departmentId = ''): [RecursiveInstitutionUnit[], boolean] => {
```
unødvendig å spesifisere type her (lærte det nettopp av eslint)